Tools/i18n/pygettext.py

```diff
@@ -204,7 +197,7 @@ def make_escapes(pass_nonascii):
     global escapes, escape
     if pass_nonascii:
         # Allow non-ascii characters to pass through so that e.g. 'msgid
-        # "H�he"' would result not result in 'msgid "H\366he"'.  Otherwise we
+        # "Höhe"' would result not result in 'msgid "H\366he"'.  Otherwise we
```

Comment:
Likewise
```suggestion
        # "Höhe"' would result not result in 'msgid "H\366he"'.  Otherwise we
```

Reply:
IIRC, you need to open/save the file as iso 8859-2

Reply:
post-merge it seems fine: https://github.com/python/cpython/blob/237f186da48d45ef75ec3add3f2ffbfac75fa281/Tools/i18n/pygettext.py#L187